src/pages/kvittering/index.tsx

```diff
@@ -66,9 +68,3 @@ export default function Kvittering(props: KvitteringProps)
         </div>
     );
 }
-
-export async function getServerSideProps() {
```

Comment:
Tror dette er nødvendig å ha med for at appen skal server-side rendres, og ikke pre-rendres. Ser ut som bygget på branch feiler nå siden window ikke er tilgjengelig på byggetid?